endpoints/openrtb2/amp_auction.go

```diff
@@ -240,6 +254,37 @@ func (deps *endpointDeps) AmpAuction(w http.ResponseWriter, r *http.Request, _ h
 		return
 	}
 
+	if isRejectErr {
+		labels, ao = rejectAmpRequest(*rejectErr, w, reqWrapper, labels, ao, errL)
+		return
+	}
+
+	labels, ao = sendAmpResponse(w, response, reqWrapper, labels, ao, errL)
+}
+
+func rejectAmpRequest(
```

Comment:
Nitpick: Could you re-format this function so the parameters are all on one line to follow the patterns of other functions in this file?

Reply:
Please clarify, do you wish us to re-format functions to one line only in files we edited or is it a whole project convention and we should use the same format across all functions we created? Because we tried to keep the line length under 120 characters long in all functions we created.

Reply:
Hi @4lexvav, the team discussed this offline and decided that we want to favor readability over consistency in this case so we prefer you stick with your approach of breaking long one-line function signatures up so that they span multiple lines. 120 characters seems like a good rule of thumb.
@AlexBVolcy, there are some cases where we already do this and we will continue to make this change throughout our code base over time.

Reply:
Sure, I restored the multiline format of function signatures.